modules/tensor_mechanics/src/materials/ComputeLinearElasticStress_CUT_DT.C

```diff
@@ -0,0 +1,50 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#include "ComputeLinearElasticStress_CUT_DT.h"
+
+template<>
+InputParameters validParams<ComputeLinearElasticStress_CUT_DT>()
+{
+  InputParameters params = validParams<ComputeStressBase>();
+  params.addClassDescription("Compute stress using elasticity for small strains");
+  return params;
```

Comment:
OK, i agree about the test.  I'll provide it as soon as i can.  Currently i'm at a conference, so i can't predict when that'll be.  Almost definitely this week though!